reproducer/src/main/java/my/example/otel/reproducer/support/CxfBeans.java

```diff
@@ -20,6 +25,13 @@
 @Configuration(proxyBeanMethods = false)
 class CxfBeans {
 
+    @PostConstruct
```

Comment:
W3C Trace + Baggage is the default propagator

Reply:
Are you sure ? By reading [1] it seems it's "no-op propagators"...

[1] https://opentelemetry.io/docs/specs/otel/context/api-propagators/#global-propagators 

"The OpenTelemetry API MUST use no-op propagators unless explicitly configured otherwise"

Reply:
https://github.com/spring-projects/spring-boot/blob/3.2.x/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java#L175

Reply:
I think that's different from the one Camel uses [1] but I may be wrong ...

[1] https://github.com/apache/camel/blob/d4792ba9226cb31de91393b10fad5d6f914600d0/components/camel-opentelemetry/src/main/java/org/apache/camel/opentelemetry/OpenTelemetryTracer.java#L147

Reply:
Exactly, you need to set it manually. You should be all set now ! Closing